BackofficeBundle/Tests/EventSubscriber/FieldTypeTypeSubscriberTest.php

```diff
@@ -160,4 +163,44 @@ public function testPreSetDataWithNoTypeSet()
         Phake::verify($this->fieldType, Phake::never())->removeOption(Phake::anyParameters());
         Phake::verify($this->form, Phake::never())->add(Phake::anyParameters());
     }
+
+    /**
+     * Test preSetData when type has no option
+     */
+    public function testPreSetDataWithTypeNoOption()
+    {
+        Phake::when($this->fieldType)->getType()->thenReturn('hidden');
+
+        $this->subscriber->preSetData($this->event);
+
+        Phake::verify($this->fieldType, Phake::times(0))->addOption(Phake::anyParameters());
```

Comment:
the verification in testPreSetDataWithTypeNoOption method and testPreSubmitWithTypeNoOption method ?